go-controller/observability-lib/model/network_event.go

```diff
@@ -2,9 +2,30 @@ package model
 
 import (
 	"fmt"
+)
+
+const (
+	// Constants are duplicated to minimize dependencies
+	// When adding constants here, add them in network_event_test.go too
 
-	libovsdbops "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdb/ops"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	// libovsdb constants: cf github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdb/ops
```

Comment:
what does "cf" mean?

Reply:
huh
I've been using it very frequently, probably borrowed from French, and you're the first one asking ... and it seems like I should stop using it lol
I use it in the meaning of "See also" ... which, according to Wiktionary, is [non-standard & proscribed](https://en.wiktionary.org/wiki/cf.#English)